maquetado/scss/detail.scss

```diff
@@ -1,6 +1,22 @@
 .card {
+  background-color: #FFF;
+  box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.5);
+  display: flex;
+  flex-direction: row;
+  height: 416px;
+  padding: 24px 24px;
+  width: 900px;
+}
+
+.photo-detail {
+  height: 368px;
+  width: 261px;
```

Comment:
Normally you don't use exact measures of invision so try to use multiples of N number for example 5, with this the width change to 260px; please make this change in all app